android/src/main/java/com/getcapacitor/community/audio/AudioAsset.java

```diff
@@ -40,6 +46,26 @@
     }
   }
 
+  AudioAsset(
+    NativeAudio owner,
+    String assetId,
+    String url,
+    int audioChannelNum,
+    float volume
+  ) throws Exception {
+    this(owner, assetId);
+
+    if (audioChannelNum < 0) {
+      audioChannelNum = 1;
```

Comment:
qq. Why do we need to use channel 1 instead of channel 0 in this case?

Reply:
because channel 0 means an empty array of audio asset according to the implementation.

Reply:
I guess that also the check is not done properly. It should be `audioChannelNum <= 0` and then set it to 1 in those cases, to guarantee having at least an AudioAsset to play. Otherwise it doesn't make sense